src/wfqc/network.py

```diff
@@ -15,46 +17,61 @@
 # TODO: import jsonpath_ng.ext      # More efficient json processing look into if actually computationally more efficient 
 import igraph               # Used to create te citationa graph 
 
+import sys
+sys.path.insert(0, os.path.abspath(os.path.join(os.getcwd(), 'src')))
 
-import wfqc.data
+import wfqc.data 
 
 
-def download_data(outpath, testSize, randomSeed, topicID):
+
+async def download_data(outpath, testSize, randomSeed, topicID):
```

Comment:
The name of the method is quite generic, documentation and param types would be quite useful here.